### DISCUSSION TITLE
How to stop italics in tree-sitter's syntax highlighting?

### DISCUSSION THREAD

**Discussion by ianyepan** [emacs-tree-sitter/elisp-tree-sitter#130] [Q&A]
First of all, thanks for porting tree-sitter to Emacs -- I only recently discovered it and it's super awesome.

## Issue
How can I stop the method calls to be italicized as in the following screenshot? These keywords were not italics before I switched to using tree-sitter for its superior syntax highlighting.

## Screenshot
![image](https://user-images.githubusercontent.com/36686900/111611345-51077280-8817-11eb-9eba-d8f9c074a797.png)

**Comment by ubolonton** [comment]
You can put the point on the highlighted text and evaluate `(get-text-property (point) 'face)` to see what's the face used. Then you can customize that face to remove the `:slant` attribute.

In this case, it's probably the face `tree-sitter-hl-face:property`.

**Reply by angrycandy** [reply]
- I couldn't get `(get-text-property (point) 'face)` to work for me.
- `M-x describe-text-properties` worked for me and I was able to use the resulting buffer to customize the attribute.

Thanks.
